examples/default/sources/e_shop_clothing.yaml

```diff
@@ -0,0 +1,2 @@
+type: "file"
+path: "data/e-shop-clothing.csv"
```

Comment:
Use:

```
type: "https"
uri: https://cdn.rilldata.com/examples/0.16/ecomm-click-stream/e-shop-clothing.csv
```

Reply:
Yeah that make sense we better not save data here, we can more examples in future with http. removing data and addressing these changes